config/webhook/manifests.yaml

```diff
@@ -1,61 +1,11 @@
 
----
-apiVersion: admissionregistration.k8s.io/v1
-kind: MutatingWebhookConfiguration
-metadata:
-  creationTimestamp: null
-  name: mutating-webhook-configuration
-webhooks:
-- admissionReviewVersions:
-  - v1
-  clientConfig:
-    service:
-      name: webhook-service
-      namespace: system
-      path: /mutate-infrastructure-cluster-x-k8s-io-v1beta1-byoclustertemplate
-  failurePolicy: Fail
-  matchPolicy: Equivalent
-  name: mbyoclustertemplate.kb.io
-  rules:
-  - apiGroups:
-    - infrastructure.cluster.x-k8s.io
-    apiVersions:
-    - v1beta1
-    operations:
-    - CREATE
-    - UPDATE
-    resources:
-    - byoclustertemplates
-  sideEffects: None
-
 ---
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
```

Comment:
Do we need to keep this empty webhook?

Reply:
We have Validating webhooks for byohost, so it's not totally empty. As per previous discussion we decided to remove webhooks for byoclustertemplates, so that's why removed them.